.github/workflows/pr_labels_check.yml

```diff
@@ -22,7 +22,7 @@ jobs:
     name: Test approved or docs
     runs-on: ubuntu-latest
     steps:
-      - uses: danielchabr/pr-labels-checker@master
+      - uses: danielchabr/pr-labels-checker@v3.0
```

Comment:
@Zejnilovic suggested it, because the master is against some standards

Reply:
Are we sure it works? Because I had tried it with this entry when implementing the action and for an unknown reason, it was saying my the action plugin cannot be found/downloaded.